### ISSUE TITLE
Set up Eclipse Maven repo CI/CD

### ISSUE THREAD

**Issue by berezovskyi** [eclipse/lyo.validation#4]
~~Waiting for https://bugs.eclipse.org/bugs/show_bug.cgi?id=525468~~~

- [x]  set up a Jenkins build (use lyo-store as a baseline or explore Jenkinsfile for fun and profit)

- [x]  deploy Javadocs (either here or via Eclipse – the latter is preferable I think)

cc @jadelkhoury @jamsden

**Comment by berezovskyi** [comment]
Okay, so the Jenkins is up and running. This issue is blocking #2 actually because Yash wants to point people in README to the library Javadocs.

**Comment by berezovskyi** [comment]
@jadelkhoury @jamsden I think a small problem is that at least the release JAR deployment has to happen on Jenkins right at the 2.3.0 final release moment together with the rest of the artefacts. Jim, if we get a green light, I might need you to do this as well – ie to create a new build config for lyo-validation using lyo-store build config as a template.

**Comment by jadelkhoury** [comment]
re blocking https://github.com/eclipse/lyo-validation/issues/2 , I don't think we should care for now. We can always publish the readme later.

**Comment by berezovskyi** [comment]
well, actually it's blocking a release, not README

**Comment by berezovskyi** [comment]
or, more precisely, both. but action is needed now because of release block

**Comment by jadelkhoury** [comment]
Let me rephrase, my point is that we should not wait on fixing #2 before we release.

**Comment by berezovskyi** [comment]
Yes, sure, we can release without #2. But this is issue #4, it's about deploying JARs to Eclipse Maven.

**Comment by jadelkhoury** [comment]
ok. I misread the issue then.
